app/gui/view/debug_scene/cloud-dashboard/src/lib.rs

```diff
@@ -37,131 +37,483 @@
+            };
+            grid.set_entries_params(params);
+            let row = model.rows();
+            let col = Columns::LEN;
+            grid.reset_entries(row, col);
+            grid.frp().focus();
+
+
+            // === Configure Simple Grid View Headers ===
+
+            let header_frp = grid.header_frp();
+            frp::extend! { network
+                requested_section <- header_frp.section_info_needed.map(|&(row, col)| {
+                    let sections_size = 2 + col;
+                    let section_start = row - (row % sections_size);
+                    let section_end = section_start + sections_size;
+                    let model = header_entry_model(section_start, col);
+                    (section_start..section_end, col, model)
+                });
+                header_frp.section_info <+ requested_section;
             }
+            std::mem::forget(network);
+
+
+            // === Configure Scrollable Grid View ===
 
-            View { frp, model }
+            // These sizes are arbitrary, because the grid will get resized as soon as the WebGL
+            // context is ready (because the shape of the viewport will change as a result).
+            grid.scroll_frp().resize(Vector2(400.0, VIEWPORT_HEIGHT));
+
+            app.display.add_child(&model.root);
+
+
+            // === Populate Table With Data ===
+
+            let client =
+                enso_cloud_http::Client::new(API_GATEWAY_ID, AWS_REGION, TOKEN.to_string());
+            get_projects(client, input.clone_ref());
+
+
+            std::mem::forget(grid);
+            std::mem::forget(input);
+            std::mem::forget(out);
+
+            Self { frp, model }
         }
     }
 
+    // FIXME [NP]: Should we be assuming that the `row, col` are always in bounds? If we do so, we
+    // can avoid having to return `Option` here, but we run the risk of panicking in the event of an
+    // out of bounds `row, col`. Though this should only happen on implementation error.
+    fn header_entry_model(row: Row, col: Col) -> EntryModel {
+        assert!(row == 0, "Header row was {row}, but it is expected to be first row in the table.");
+
+        let column = match Columns::from_discriminant(col) {
+            Some(column) => column,
+            None => panic!(
+                "Attempted to display entry at column {col}, but we the table only has {} columns.",
+                Columns::LEN
+            ),
+        };
+
+        EntryModel {
+            // FIXME [NP]: Columns should not only render their name, but also their sorting,
+            // represented by an arrow icon.
+            text:           column.to_string().into(),
+            disabled:       Immutable(true),
+            override_width: Immutable(if col == 1 && row == 5 { Some(180.0) } else { None }),
+        }
+    }
+
+    fn get_projects(
+        client: enso_cloud_http::Client,
+        input: crate::projects_grid::api::public::Input,
```

Comment:
You can pass here `set_projects: frp::Source<...>` and then you'd need to do `set_projects.emit(...)`.

Reply:
when I tried this I got:
```
    Checking debug-scene-cloud-dashboard v0.1.0 (/home/indiv0/src/enso/app/gui/view/debug_scene/cloud-dashboard)
error[E0308]: mismatched types
   --> app/gui/view/debug_scene/cloud-dashboard/src/lib.rs:554:30
    |
554 |         get_projects(client, input.set_projects);
    |         ------------         ^^^^^^^^^^^^^^^^^^ expected struct `enso_frp::SourceData`, found struct `enso_frp::AnyData`
    |         |
    |         arguments to this function are incorrect
    |
    = note: expected struct `enso_frp::stream::WeakNode<enso_frp::SourceData<std::rc::Rc<std::vec::Vec<enso_cloud_view::project::Project>>>>`
               found struct `enso_frp::stream::WeakNode<enso_frp::AnyData<std::rc::Rc<std::vec::Vec<enso_cloud_view::project::Project>>>>`
note: function defined here
   --> app/gui/view/debug_scene/cloud-dashboard/src/lib.rs:581:8
    |
581 |     fn get_projects(
    |        ^^^^^^^^^^^^
582 |         client: enso_cloud_http::Client,
583 |         set_projects: frp::Source<Rc<Vec<view::project::Project>>>,
    |         ----------------------------------------------------------

For more information about this error, try `rustc --explain E0308`.
error: could not compile `debug-scene-cloud-dashboard` due to previous error
```
How do I convert `AnyData`-marked `Source` to a `SourceData`-marked `Source`?

Reply:
I don't see these lines in this file anymore, where are they now?

Reply:
`get_projects` is now in the `projects_grid` module